engine/api/action/builtin.go

```diff
@@ -56,6 +56,11 @@ Parse given file to extract coverage results.`
 		Description: `Path of the coverage report file.`,
 		Type:        sdk.StringParameter,
 	})
+	cover.Parameter(sdk.Parameter{
+		Name:        "minimum",
+		Description: `Minimum percentage of coverage required (-1 means no minimum).`,
+		Type:        sdk.StringParameter,
```

Comment:
maybe it should be an advanced parameter

Reply:
I don't know what an advanced parameter is :/
Do you have any pointer on this?

Reply:
done